### PR TITLE
[#160726559] Admin should view data pertaining to their center

### PR DESCRIPTION
**Pull request by JoanNgatia**
## What does this PR do?
`Restrict an admin to view assets and Office data from their location only.`

## Description of the task to be completed:
-  Define Location field on Viewsets for OfficeBlock, OfficeFloor, OfficeFloorSections, and OfficeWorkspace models
-  Define Location field on Viewsets for Assets, Asset

## Relevant pivotal tracker stories:
#160726559

### CODE REVIEW

**Comment by egichuri** [comment]
@JoanNgatia looks good.. only that failing test

**Comment by esirK** [comment]
@JoanNgatia  My guess is that the tests are failing since the user we are using does not have a location set for him/her thus the following [line](https://github.com/AndelaOSP/art-backend/pull/215/files#diff-265524153427c5c3f0dd9ae087f16a4bR75) returns an empty queryset Providing a default location for the user in the tests might work. I will test this out and let you know what I find.

File: api/views.py [hunk 1]
```diff
@@ -64,15 +64,19 @@ class UserViewSet(ModelViewSet):
...
-    queryset = Asset.objects.all()
     permission_classes = [IsAuthenticated, IsAdminUser]
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ['get', 'post', 'put', 'delete']
     filter_backends = (filters.DjangoFilterBackend,)
     filterset_class = AssetFilter
 
+    def get_queryset(self):
+        user_location = self.request.user.location
+        queryset = Asset.objects.filter(asset_location=user_location)
```
[the remark this answers is not in the document]
Comment (egichuri) [reply api/views.py]: good catch @esirK 
I think we can return an empty queryset if the user has no location set

**Comment by egichuri** [comment]
changes moved [here](https://github.com/AndelaOSP/art-backend/pull/224)

**Pull request closed without merging**